client-v2/src/actions/Collections.ts

```diff
@@ -2,7 +2,7 @@ import uniq from 'lodash/uniq';
 import { batchActions } from 'redux-batched-actions';
 import {
   getArticlesBatched,
-  getCollection as fetchCollection,
+  getCollections as fetchCollection,
```

Comment:
Tiny tiny thing but maybe just rename this to `fetchCollections` it confused me for longer than it should have 😊

Reply:
Definitley -- I think there's another PR to write that renames all of our 'gets' to 'fetches'!